lib/perm/can-order-for.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var _ = require('lodash');
+
+
+function canOrderFor (args, cb) {
+  const seneca = this;
+  const plugin = args.role;
+  const userId = args.user.id;
+  const applications = args.params.applications;
+
+  seneca.act({role: 'cd-profiles', cmd: 'load_children_for_user', userId: userId, user: args.user },
+    function(err, children) {
+      const childIds = children.map(child => child.userId);
```

Comment:
Missing error handling

Reply:
As well, load_children_for_user returns a "soft" error (https://github.com/CoderDojo/cp-users-service/blob/master/lib/profiles/load-children-for-user.js#L10) that should be considered as "falsy" for a permission handler